pandas/tests/extension/test_integer.py

```diff
@@ -224,6 +224,10 @@ def test_value_counts(self, all_data, dropna):
 
         self.assert_series_equal(result, expected)
 
+    @pytest.mark.skip(reason="uses nullable integer")
```

Comment:
maybe at a GH ref back to this thread?  it isnt obvious to a reader why this message is relevant